packages/api/src/schema/schema.yaml

```diff
@@ -1221,11 +1221,11 @@ components:
                       readOnly: true
                       description:
                         URL to access file through HTTP on an IPFS gateway
-                    erc1155MetadataCid:
```

Comment:
nit: What about always use `CID` in uppercase, so erc1155MetadataCID?

Reply:
* nftMetadataCID

Reply:
Hmm are you reviewing an outdated version? Here it is `nftMetadataCid` already

Regarding casing, I'd prefer to keep it consistent. In Go, the pattern is to keep acronyms all upper or all lowercase. I'm not sure if there is an officially recommended pattern for node/JSON, but we already have things like `playbackId` (not `ID`), `recording/playbackUrl` on our schemas, so I'd prefer to keep it consistent.